code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1455,6 +1455,9 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 						var/datum/antagonist/rev/rev = H.mind.has_antag_datum(/datum/antagonist/rev)
 						if(rev)
 							rev.remove_revolutionary(FALSE, user)
+						var/datum/antagonist/hivevessel/woke = H.is_wokevessel()
+						if(woke)
```

Comment:
Why did you remove the second conditional? That one was actually needed. Now it can unconvert brainwashed people before one mind even happens.

Reply:
intentional, at least it was half an hour ago
fixin' aaaa